did/plugins/google.py

```diff
@@ -301,6 +304,7 @@ def __init__(self, option, name=None, parent=None, user=None):
             apps = [app.lower() for app in split(config["apps"])]
         except KeyError:
             apps = DEFAULT_APPS
+        self.skip = config.get("skip")
```

Comment:
```suggestion
        self.skip = config.get("skip", [])
```

To make it working when no `skip` is present.